packages/fxa-settings/src/components/ConnectedServices/index.tsx

```diff
@@ -314,7 +314,7 @@ export const ConnectedServices = () => {
               >
                 <p
                   id="connected-devices-sign-out-description"
-                  className="my-4 text-center"
+                  className="my-4 text-center break-all"
                 >
                   Your browsing data will remain on your device (
                   {client.current!.name}
```

Comment:
sorry! I haven't commit my changes. I have already added the suggestions from  @LZoog. I will commit the new version now.

Reply:
@jodyheavener I have commit my last changes, but it seems that I have messed up something. I will try to fix it and get back to you with questions if I get stuck

Reply:
When I was working on my issue for a while, I noticed that some things have changed in the file in the meantime which also affected my changes (names of variables have changed). I made this observation after updating my main branch. After that I also updated the branch on which I edited my issue with git fetch. Now I wanted to commit my changes, and found that a bunch of files were staged that I had nothing to do with (probably all files that have changed since then). But since I only wanted to push my changed file, I ran git unstage --all and then only committed and pushed the one file. However, on Github, the commit of that file shows me other changes as well. 

Now I wonder if it makes more sense to just create a new branch and make a new pull request instead of continuing to work on this branch. 

Please excuse the inconvenience

Reply:
@jodyheavener I fixed it by using `git rebase` instead of `git merge`. That was my first time I had to handle merge conflicts 😃